communitiesuk/funding-service-design-application-store
# Added seed application config for cof r3w2

https://dluhcdigital.atlassian.net/browse/FS-3427

### Description
Added seed application config for cof r3w2

### How to test
From VSCode, select `"Seed Applications in application-store"` in launch config & run. In the interactive prompt select `COF R3W2`.
(or)
From container terminal, `docker exec -it <container_id> bash`, excute the script `python scripts/seed_db_test_data.py` will launch interactive prompt select `COF R3W2` and run

Observe `COF R3W2` is seeded on frontend for the provided account .
